app/assets/javascripts/controllers/CommonChatController.js

```diff
@@ -298,4 +313,21 @@ export default class CommonChatController {
         this.showEndChatPage(true);
     }
 
+    onPostChatSurveyDigitalAssistantSubmitted(surveyPage) {
+        const answers = {
+            answers: [
+                { id: getRadioId("q1-"), text: getRadioValue("q1-"), freeform: false },
+                { id: "q2-", text: getTextAreaValue("q2-"), freeform: true },
+                { id: getRadioId("q3-"), text: getRadioValue("q3-"), freeform: false },
+                { id: "q4-", text: getTextAreaValue("q4-"), freeform: true }
+            ]
+        };
+
+        var surveyWithAnswers = Object.assign(answers, digitalAssistantSurvey);
+
+        this._sendPostChatSurveyDigitalAssistant(this.sdk).submitPostChatSurvey(surveyWithAnswers, automaton, timestamp);
+        surveyPage.detach();
+        this.showEndChatPage(true);
+    }
+
 };
```

Comment:
Can we add New line at the end of the file please?

Reply:
Done